packages/cli/src/cog/cog.stac.job.ts

```diff
@@ -191,9 +196,11 @@ export class CogStacJob implements CogJob {
 
         const sourceProj = job.sourcePtms.proj;
 
-        const bbox = sourceProj.boundsToWgs84BoundingBox(
-            metadata.bounds.map((a) => Bounds.fromJson(a)).reduce((sum, a) => sum.union(a)),
-        );
+        const bbox = [
+            sourceProj.boundsToWgs84BoundingBox(
```

Comment:
Does this break with the antimeridian?

is the reason their are multiple bbox's for the antimeridian?

Reply:
No. only one bbox is needed event when AM is crossed; it just means the first longitude will be greater than the second longitude

Reply:
Hi Guys, it's best practise to split across the anti-meridian for interoperability reasons with clients. See https://tools.ietf.org/html/rfc7946#section-3.1.9

Reply:
@jacott with Jeremy's comment maybe we should split it on the antimeridian? Does the stac spec tell us not to?

Reply:
@blacha , @palmerj The GeoJSON RFC does not split `bbox`. STAC uses the GeoJSON for items but I defer to others and will split if you think I should.

Reply:
What the link above is for is polygons which we do split.

Reply:
Ok thanks. You are correct. As long as we split the footprint geometry then we are ok. Clients querying on the bbox via the STAC API will of course need to normalise their input bboxes